chemicaltoolbox/autodock_vina/prepare_ligands/.shed.yml

```diff
@@ -0,0 +1,10 @@
+categories:
+- Computational chemistry
+description: Software for protein-compound docking - the prepare ligands step
+long_description: |
+    Splits a multi-molecucles file, such as a SD file, into single MOL2 format files that can be
+    individually docked.
+name: autodock_vina_prepare_ligands
+owner: tdudgeon
```

Comment:
Sorry, that shed definition was not supposed to be there. Originally I started this in the autodock_vina dir but moved to to openbabel once it became clear this was really a obabel tool.
SO I've deleted that .shed.yml file.

Reply:
Sorry, that .shed.yml file should not have been there. I started the work in the autodock_vina dir but moved it to openbabel when it because clear this was really an obabel tool.  File has been deleted.

Reply:
Maybe in time we'll have a `docking` category of which `autodock_vina` is one sub-type, in which case this tool might well be best place in that `docking` dir.
For now I leave it in `openbabel` as there doesn't seem a better home.